src/good_fences_runner.rs

```diff
@@ -88,22 +89,10 @@ impl GoodFencesRunner {
 
     pub fn find_import_violations<'a>(
         &'a self,
-        ignored_dirs: Option<Vec<String>>,
+        ignored_dirs: Vec<regex::Regex>,
```

Comment:
This parameter can also be deleted, since its only used to plumb through to evaluate_fences